airbyte-integrations/connectors/source-google-analytics-data-api/unit_tests/test_streams.py

```diff
@@ -88,30 +88,36 @@ def test_request_body_json(patch_base_class):
         ],
         "dateRanges": [request_body_params["stream_slice"]],
         "returnPropertyQuota": True,
+        "offset": "0",
+        "limit": "10000"
     }
 
     request_body_json = GoogleAnalyticsDataApiBaseStream(authenticator=MagicMock(), config=patch_base_class["config"]).request_body_json(**request_body_params)
     assert request_body_json == expected_body_json
 
 
 def test_next_page_token_equal_chunk(patch_base_class):
-    stream = GoogleAnalyticsDataApiBaseStream(authenticator=MagicMock(), config=patch_base_class["config"])
+    additional_config = {
+        "offset": "0",
+        "limit": "10000"
```

Comment:
use `integer` instead of `string`

Reply:
Change all occurrences limit and offset to integer type :heavy_check_mark: